src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +17,19 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		String figure = e.getActionCommand().toLowerCase();
+		switch (figure){
+			case "figure joe 1":
+				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+				break;
+
+			case "figure joe 2":
+				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+				break;
+
+			case "jane":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.